### DISCUSSION TITLE
user_name = 'nobody' breaks query_log?

### DISCUSSION THREAD

**Discussion by oliebe** [DNSCrypt/dnscrypt-proxy#2051] [Q&A]
I tried to enable query_log but it was not working until i commented the line user_name = 'nobody'. Is this supposed to happen? The main log file always worked with this uncommented.

**Comment by jedisct1** [comment]
Does user `nobody` have write access to the directory you are trying to store logs in?

**Reply by oliebe** [reply]
```shell
$ sudo -u dnscrypt test -w /var/log/dnscrypt-proxy/ ; echo "$?"
0
$ sudo -u nobody test -w /var/log/dnscrypt-proxy/ ; echo "$?"
1
$ sudo -u root test -w /var/log/dnscrypt-proxy/ ; echo "$?"
0
```

Nope
Just find it weird that the dnscrypt-proxy.log works fine with `user_name=nobody` but the query.log and others don't. 
In my head the dnscrypt-proxy.log should behave just like the other logs
